src/parser/mage/fire/CHANGELOG.js

```diff
@@ -1,8 +1,20 @@
-// import React from 'react';
+import React from 'react';
 
 import { Sharrq } from 'CONTRIBUTORS';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
 
 export default [
+  {
+    date: new Date('2018-11-15'),
+    changes: <>Added support for <SpellLink id={SPELLS.BLASTER_MASTER.id} />.</>,
+    contributors: [Sharrq],
+  },
+  {
```

Comment:
Something wrong with the diff here, you have two changelog entries for the hot streak changes